test/supports/mock_env_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module MockEnvHelper
+  def mock_env(partial_env_hash)
+    old = ENV.to_hash
+    ENV.update partial_env_hash
+    begin
+      yield
+    ensure
+      ENV.replace old
+    end
+  end
```

Comment:
```suggestion
  def mock_env(hash_for_mock)
    env_hash = ENV.to_hash.merge(hash_for_mock)
    ENV.stub(:[], ->(key) { env_hash[key] }) do
      yield
    end
  end
```

ENVを書き替えるのは事故が怖いのでstubメソッドのした方が安心かも

Reply:
こちらで修正しました。
[f546eb4](https://github.com/fjordllc/bootcamp/pull/6970/commits/f546eb451dd35777758efb5c09b75772102b433a)
rubocopでこの警告が出たため、yieldを使わない形に書き換えています。
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/ExplicitBlockArgument